web/models.py

```
@@ -2008,7 +2011,23 @@
+        SESSION = "session", "Session Exam"
+        COURSE = "course", "Course Exam"
+        QUIZ = "quiz", "Quiz"
+
+    exam_type = models.CharField(
+        max_length=10,
+        choices=ExamType.choices,
+        default=ExamType.QUIZ,
+    )
+
     STATUS_CHOICES = [("draft", "Draft"), ("published", "Published"), ("private", "Private")]
+    EXAM_TYPES: ClassVar[List[Tuple[str, str]]] = [
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)